R/shiny_helpers.R

```diff
@@ -0,0 +1,9 @@
+#' helper function for shiny launch
+#' @export
+#' @importFrom mda.streams get_meta
+#' @keywords internal
+get_types <- function(){
```

Comment:
so in this case 'types' is not the list of meta_types but the list of data columns within any of the metadata tables? naming is a little confusing to me